modules/kubernetes/control-plane-logs/README.md

```diff
@@ -0,0 +1,49 @@
+# Controle plan logs
+
+This module is used to run a small [vector](https://vector.dev/) deployment inside the cluster.
+It listens to a message queue, parses the output and pushes it to stdout.
+
+vector.toml includes the config how to connect to it's different endpoints.
+Vector supports unit testing, and to verfiy it's config you can run `vector test vector.toml`.
+
+If you need to update the vector.toml you need to perform the following
+manuall task to regenerate the secret located in the templates file
+cat vector.toml |base64 -w 0
+All the variabels in vector.toml is read in through environment variabels in the deployment.
```

Comment:
Do we think this is an okay way of working?

Reply:
Make it a configmap instead of a secret